diagnostic.md

```diff
@@ -40,31 +76,39 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
 ```sh
-  # < Your Response Here >
+  # The serializer filters the information provided within API responses.
 ```
 
 ```rb
 class ProfileSerializer < ActiveModel::Serializer
+  attributes :id, :given_name, :surname, :email, :favorites
 end
 ```
 
 1.  What would the command be to _scaffold_ out a **join table** for Favorites from
 the above `Movies` and `Profiles`.
 
 ```sh
-  # < Your Response Here >
+  # rails g scaffold favorites profile:references movie:references
 ```
 
 1.  What is `Dependent: Destroy` and where/why would we use it?
 
 ```sh
-  # < Your Response Here >
+  # Dependent: Destroy means if you want to destroy an item in a table, you
+  # must first destroy all references to it within joined tables.  In this example,
+  # if you want to destroy a given profile or movie, you first need to remove
+  # all instances in the favorites table that reference that profile or movie.
 ```
```

Comment:
It just means that when we use it, it deletes the instance of that class in the model, AND all associated objects as well.